DeviceAdministration/Infrastructure/Properties/AssemblyInfo.cs

```diff
@@ -33,7 +33,4 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
-
-// Allow access to assembly internals for unit testing
-[assembly: InternalsVisibleTo("Infrastructure.UnitTests")]
```

Comment:
why did we remove this ? @handeadiguzel I remember you changing some internals to public for unit testing.